pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, fetchFromGitLab , which, go-md2man, coreutils, substituteAll }:
```

Comment:
Please be consistent in spacing: `{ stdenv, fetchFromGitLab , which, go-md2man, coreutils, substituteAll }:`